server/src/main.rs

```diff
@@ -207,7 +209,8 @@ async fn main() -> Result<(), anyhow::Error> {
                             .start_streaming_delta_background_task(
                                 ClientMetaInformation {
                                     app_name,
-                                    instance_id,
+                                    instance_id: identifier.clone(),
```

Comment:
it looks like streaming never respected user provided instance_id and always took our_instance_data_for_app_context.idnetifier

Reply:
indeed.